runelite-client/src/main/java/net/runelite/client/util/ImageCapture.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2019, Alexsuperfly <https://github.com/Alexsuperfly>
```

Comment:
While you can include your copyright on this since you adapted it, it should also include the copyright of the original author. (looks like lotto?)